finder-app/finder-test.sh

```diff
@@ -45,12 +45,35 @@ fi
 #make clean
 #make 
 
+path=$(which writer)
+
+if [ ${path} = "/usr/bin/writer" ] 
+then 
+	WRITER_PATH=writer
+else
+	WRITER_PATH=./writer
```

Comment:
@jasan2303 what if writer is at some other path than ./ or /usr/bin ? what is the purpose of which utility usage here?